server/routes/discover.ts

```diff
@@ -63,6 +63,45 @@ discoverRoutes.get('/movies', async (req, res) => {
   });
 });
 
+discoverRoutes.get<{ language: string }>(
+  '/movies/language/:language',
+  async (req, res) => {
+    const tmdb = createTmdbWithRegionLanaguage(req.user);
+
+    const languages = await tmdb.getLanguages();
+
+    const language = languages.find(
+      (lang) => lang.iso_639_1 === req.params.language
+    );
```

Comment:
We should probably throw an error here if there is no matching language?

Reply:
@sct Do we want to also throw errors for the genre/studio/network Discover routes if we can't find matches for the IDs?